serendipity_event_vgwort/serendipity_event_vgwort.php

```diff
@@ -46,8 +46,8 @@ function setupDB() {
         global $serendipity;
         $sql = "CREATE TABLE  IF NOT EXISTS {$serendipity['dbPrefix']}vgwort (
                     entry_id INTEGER,
-                    counter_public TEXT UNIQUE NOT NULL,
-                    counter_private TEXT UNIQUE NOT NULL);";
+                    counter_public VARCHAR(32) UNIQUE NOT NULL,
```

Comment:
This  change I do not like. What if the counter gets bigger? Why not just keep it a TEXT data type?

Reply:
It's not possible to use TEXT and UNIQUE in MySQL, see e.g. https://techjourney.net/mysql-error-1170-42000-blobtext-column-used-in-key-specification-without-a-key-length/

I think it's more important to keep UNIQUE than the TEXT type?